frontend/src/metabase/query_builder/components/expressions/Editor/custom-tooltip.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  type EditorState,
+  type Extension,
```

Comment:
The popup sometimes closes when I try to click "Learn more", see 0:17

https://github.com/user-attachments/assets/bd922fc2-6f9d-4c78-bbae-4279fe73e028

Reply:
I've pushed a fix for this. Please have another look.

Reply:
So much better :+1: Now I can also select text in the popover :tada: